### ISSUE TITLE
regexp: Go's regex is even slower than Python

### ISSUE THREAD

**Issue by gophobic** [ksimka/go-is-not-good#48]
https://github.com/golang/go/issues/26623

**Comment by onlyjob** [comment]
True... Native Regex and Strings are ridiculously slow in Golang... :(

**Comment by cznic** [comment]
Depends on your bias. If you care more about the worst case, which you should for obvious [security reasons](https://en.wikipedia.org/wiki/ReDoS), PCRE (and thus Java, Perl, PHP, Python, Ruby, ...) can be [million times slower than Go](https://swtch.com/~rsc/regexp/regexp1.html) processing a "huge" 29-character string.

**Comment by onlyjob** [comment]
No bias, jest pragmatic measurements - I did my own testing and arrived at the same conclusion.
Performance does not depend on bias.
By the way nice information about _ReDoS_, thanks.
